website/static/js/atjsConfig.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * At.js config.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+
+require('Caret.js');
+require('At.js');
+
+// preventing unexpected contenteditable behavior
+
+// prevent pasting HTML
+var onPaste = function(e) {
```

Comment:
A few of the functions in atjsConfig.js are approximately 60% from elsewhere. Should I document those locations?